src/Services/Azure/Authentication/CustomChainedCredential.cs

```diff
@@ -119,4 +119,52 @@ private static DefaultAzureCredential CreateDefaultCredential(string? tenantId)
 
         return new DefaultAzureCredential(defaultCredentialOptions);
     }
+
+    private static TokenCredential? CreateVsCodeBrokerCredential(string? tenantId)
+    {
+        // VS Code Azure extension client ID
+        const string vsCodeClientId = "aebc6443-996d-45c2-90f0-388ff96faa56";
+        string authRecordPath = Path.Combine(
+            Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+            ".azure",
```

Comment:
The export code allows using either .azure and .Azure directory. Should this code check both kinds of locations?

https://github.com/microsoft/vscode-azureresourcegroups/blob/58822ce9ce5fb1fe4d654a55a6d0eb53117fc656/src/exportAuthRecord.ts#L147

Reply:
yeap, that is now added.